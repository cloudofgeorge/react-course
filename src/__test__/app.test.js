import React from 'react';
import { render } from '@testing-library/react';
import { App } from '../app';

describe('<App>', () => {
  it('should match snapshot', async () => {
    const result = render(<App />);
    expect(result.container).toMatchSnapshot();
  });
});
