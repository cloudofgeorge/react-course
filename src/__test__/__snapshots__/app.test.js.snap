// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App> should match snapshot 1`] = `
<div>
  <div
    class="layout"
  >
    <header
      class="header"
    >
      <div
        class="wrapper wrapper"
      >
        <div
          class="leftSide"
        >
          <div
            class="logo"
          >
            <a
              class="link"
              href="/"
            >
              <img
                alt="logo"
                src="/logo192.png"
                width="100%"
              />
            </a>
          </div>
          <div
            class="navigation"
          >
            <a
              href="/catalog"
            >
              Catalog
            </a>
          </div>
        </div>
        <div>
          <div
            class="cart"
          >
            <button
              data-testid="cart-button"
              disabled=""
            >
              cart 
            </button>
          </div>
        </div>
      </div>
    </header>
    <div
      class="wrapper mainContent"
    >
      <main>
        <h1>
          Welcome to our shop
        </h1>
      </main>
    </div>
    <footer
      class="footer"
    >
      <div
        class="wrapper"
      >
        footer content
      </div>
    </footer>
  </div>
</div>
`;
